CMakeLists.txt

```diff
@@ -98,6 +98,12 @@ If(NOT _HAS_CXX11_FLAG)
 EndIf()
 
 # Load some basic macros which are needed later on
+find_package(FairCMakeModules 0.1 CONFIG)
+if(NOT FairCMakeModules_FOUND)
+  message(WARNING "Please highly consider installing FairCMakeModules!")
```

Comment:
```suggestion
  message(WARNING "Could not find FairCMakeModules, falling back to local copy. It is recommended to install https://github.com/FairRootGroup/FairCMakeModules!")
```

Reply:
Maybe also consider making it a `STATUS` message, `WARNING` is quite "loud".

Reply:
Will take the better message!

Well, I would like to remove the fallback in 19.2. So I want to be loud in the 19.0 release.